include/libopencm3/stm32/l4/exti.h

```diff
@@ -0,0 +1,41 @@
+/** @defgroup exti_defines EXTI Defines
+ *
+ * @brief <b>Defined Constants and Types for the STM32F4xx External Interrupts
+ * </b>
+ *
+ * @ingroup STM32F4xx_defines
```

Comment:
Shouldn't this say `STM32L4xx_defines`? Same a few lines above and in the syscfg header.

Reply:
Yeah, just copied :D Will fix